app/routes/goal_routes.py

```diff
@@ -1 +1,100 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, request, abort, make_response
+from ..db import db
+from ..models.goal import Goal
+from ..models.task import Task
+from .route_utilities import validate_model
+
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+    try:
+        new_goal = Goal.from_dict(request_body)
+    except KeyError:
+        return jsonify({"details": "Invalid data"}), 400
+
+    db.session.add(new_goal)
+    db.session.commit()
+
+    return jsonify(new_goal.to_dict()), 201
```

Comment:
Can you tell me more about where the syntax using `jsonify` came from? 

In older versions of Flask, `jsonify` used to be required to convert python data structures to JSON. `jsonify` is no longer required and should not be used when we are sending back simple data types, dictionaries, or lists. 

I would like to see you revisit this project to remove all usage of `jsonify`. Please use hello-books and/or solar-system as reference to follow best practices and make our return values more concise.